go.mod

```diff
@@ -33,6 +33,14 @@ require (
 	golang.org/x/text v0.15.0
 )
 
+require (
```

Comment:
Merge this with the upper block IMO

Reply:
I've just used `go mod edit -replace` and `go mod tidy`, stupid tools doing stupid things :P